cypress/tests/integration/Plugins.spec.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file cypress/tests/integration/Plugins.spec.js
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ */
+
+ describe('Plugins tests', function() {
+    it('Check if Plugins.xml file was cached', function() {
+        cy.login('dbarnes', null, 'publicknowledge');
+
+        cy.get('a:contains("Website")').click();
+        cy.waitJQuery();
+        cy.get('button#plugins-button').click();
+        cy.get('button#pluginGallery-button').click();
+        cy.wait(10);
```

Comment:
Rather than using a 10-second wait, could you set an assertion-based guard on the plugin list loading properly?